.github/workflows/main.yml

```diff
@@ -151,6 +154,8 @@ jobs:
   build-matlab:
     name: Matlab toolbox for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
+    # Run this job even if post-pending-status was skipped
+    if: always()
```

Comment:
These should probably be `success() || cancelled()` so they don't run if the previous step actually failed.

Reply:
Is skipped the same as cancelled?

Even so, if the "post-pending-status" job fails, I think I'd still rather go ahead with the actual package build.

Reply:
Yeah, I'm pretty sure that's the case. Anyways, your reasoning seems fine, it's good with me to keep it this way.

Reply:
Yeah, I'm pretty sure that's the case. Anyways, your reasoning seems fine, it's good with me to keep it this way.